cmake/projects/Boost/schemes/url_sha1_boost.cmake.in

```diff
@@ -46,7 +46,7 @@ boost_component(chrono 1.47.0)
 boost_component(container 1.56.0)
 boost_component(context 1.51.0)
 boost_component(coroutine 1.53.0)
-boost_component(coroutine2 1.60.0)
+#boost_component(coroutine2 1.60.0)
```

Comment:
This function designed to work with old versions too. If "coroutine2" component removed then we should extent `boost_component` function to work not only with "since" but plus with "until". Or put it under condition `if(<boost-version> VERSION_LESS 1.65)`.